.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "roles/galaxyprojectdotorg.galaxy-tools"]
 	path = roles/galaxyprojectdotorg.galaxy-tools
-	url = https://github.com/galaxyproject/ansible-galaxy-tools.git
+	url = https://github.com/mvdbeek/ansible-galaxy-tools.git
```

Comment:
ansible-galaxy-tools is forked in ARTbio repo. Why not relying on your fork (if rolling back the changeset revision to the one you are providing is required, it is not a problem to me)